chapters/operatorsandexpressions.tex

```diff
@@ -1533,7 +1533,7 @@ \subsection{Discrete-Time Expressions}\label{discrete-time-expressions}
 \end{lstlisting}
 \end{example}
 
-\subsection{Continuous-Time Expressions}\label{continuous-time-expressions}
+\subsection{Continuous-Time and Non-Discrete-Time Expressions}\label{continuous-time-expressions}
 
-All expressions are continuous-time expressions\index{continuous-time!expression}\index{expression variability!continuous-time} including constant, parameter and discrete expressions.
-The term \firstuse{non-discrete-time expression}\index{non-discrete-time expression}\index{expression variability!non-discrete-time} refers to expressions that are neither constant, parameter nor discrete-time expressions.
+All expressions are continuous-time expressions\index{continuous-time!expression}\index{expression variability!continuous-time} including constant, parameter and discrete-time expressions.
+The term \firstuse{non-discrete-time expression}\index{non-discrete-time expression}\index{expression variability!non-discrete-time} refers to expressions that are neither constant, parameter nor discrete-time expressions.  For example, a continuous-time variable (see \cref{component-variability-prefixes-discrete-parameter-constant}) is a non-discrete-time expression.
```

Comment:
Wouldn't it make sense to be more concrete? Something like: For example, \lstinline!time! is a continuous-time built-in variable (see \cref{component-variability-prefixes-discrete-parameter-constant}) and \lstinline!time+1! is a non-discrete-time expression. Note that plain \lstinline!time! may depending on context refer to the  continuous-time variable or the non-discrete-time expression.

Oh, and is that two sentences on one line?

Reply:
Changed via suggestion by @HansOlsson.  Resolving.